# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../evas/evas-1.0.0.beta-X11-check_engine.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix X11 related configure test by using the X11 library path

--- evas-1.0.0.beta/m4/evas_check_engine.m4.orig	2010-10-04 14:21:05.496000908 +0200
+++ evas-1.0.0.beta/m4/evas_check_engine.m4	2010-10-04 14:59:09.580000847 +0200
@@ -34,11 +34,11 @@ AC_PATH_XTRA
 AC_CHECK_HEADER([X11/X.h], [have_dep="yes"])
 
 if test "x${have_dep}" = "xyes" ; then
-   AC_CHECK_LIB([X11], [XCreateImage], [have_dep="yes"], [have_dep="no"])
+   AC_CHECK_LIB([X11], [XCreateImage], [have_dep="yes"], [have_dep="no"], -L${x_libraries})
 fi
 
 if test "x${have_dep}" = "xyes" ; then
-   AC_CHECK_LIB([Xext], [XShmCreateImage], [have_dep="yes"], [have_dep="no"])
+   AC_CHECK_LIB([Xext], [XShmCreateImage], [have_dep="yes"], [have_dep="no"], -L${x_libraries})
 fi
 
 if test "x${have_dep}" = "xyes" ; then
@@ -80,15 +80,15 @@ AC_CHECK_HEADERS([X11/X.h X11/extensions
    [have_dep="no"])
 
 if test "x${have_dep}" = "xyes" ; then
-   AC_CHECK_LIB([X11], [XCreateImage], [have_dep="yes"], [have_dep="no"])
+   AC_CHECK_LIB([X11], [XCreateImage], [have_dep="yes"], [have_dep="no"], -L${x_libraries})
 fi
 
 if test "x${have_dep}" = "xyes" ; then
-   AC_CHECK_LIB([Xext], [XShmCreateImage], [have_dep="yes"], [have_dep="no"])
+   AC_CHECK_LIB([Xext], [XShmCreateImage], [have_dep="yes"], [have_dep="no"], -L${x_libraries})
 fi
 
 if test "x${have_dep}" = "xyes" ; then
-   AC_CHECK_LIB([Xrender], [XRenderCreatePicture], [have_dep="yes"], [have_dep="no"])
+   AC_CHECK_LIB([Xrender], [XRenderCreatePicture], [have_dep="yes"], [have_dep="no"], -L${x_libraries})
 fi
 
 if test "x${have_dep}" = "xyes" ; then
@@ -130,7 +130,7 @@ AC_CHECK_HEADERS([GL/gl.h GL/glx.h X11/X
    [have_dep="no"])
 
 if test "x${have_dep}" = "xyes" ; then
-   AC_CHECK_LIB([X11], [XCreateColormap], [have_dep="yes"], [have_dep="no"])
+   AC_CHECK_LIB([X11], [XCreateColormap], [have_dep="yes"], [have_dep="no"], -L${x_libraries})
 fi
 
 if test "x${have_dep}" = "xyes" ; then
@@ -709,11 +709,11 @@ AC_PATH_XTRA
 AC_CHECK_HEADER([X11/X.h], [have_dep="yes"])
 
 if test "x${have_dep}" = "xyes" ; then
-   AC_CHECK_LIB([X11], [XCreateImage], [have_dep="yes"], [have_dep="no"])
+   AC_CHECK_LIB([X11], [XCreateImage], [have_dep="yes"], [have_dep="no"], -L${x_libraries})
 fi
 
 if test "x${have_dep}" = "xyes" ; then
-   AC_CHECK_LIB([Xext], [XShmCreateImage], [have_dep="yes"], [have_dep="no"])
+   AC_CHECK_LIB([Xext], [XShmCreateImage], [have_dep="yes"], [have_dep="no"], -L${x_libraries})
 fi
 
 if test "x${have_dep}" = "xyes" ; then
